docs/components.rst

```diff
@@ -0,0 +1,76 @@
+How to use components
+=====================
+
+
+So you've seen the new fancy buttons and want to give them a try, but dont know where to start. No problem!
+
+First lets cover the basics. Discord messages can have *components*, such as buttons, dropdowns etc. These components sit in whats called an "action row". An action row can hold 5 components at a time. Your message can have 5 action rows, so thats a total of 25 components!
+
+Sending some components
+_______________________
+
+First we need to create some buttons, lets put them in a list for now. We'll use :meth:`create_button() <discord_slash.utils.manage_components>` to create a green button
+
+.. code-block:: python
+
+    from discord_slash.utils import manage_components
+    from discord_slash.model import ButtonStyle
+
+    buttons = [
+                manage_components.create_button(
+                    style=ButtonStyle.green,
+                    label="A Green Button"
+                )
+              ]
+
+So we have a button, but where do we use it. Let's create an action row with :meth:`create_actionrow() <discord_slash.utils.manage_components>` and put our buttons in it
+
+.. code-block:: python
+
+    [...]
+    action_row = manage_components.create_actionrow(*buttons)
+
+
+Fantastic, we now have an action row with a green button in it, now lets get it sent in discord
+
+.. code-block:: python
+
+    await ctx.send("My Message", components=[action_row])
```

Comment:
I think it should be noted that sending buttons work both for slash (interction) context and for any d.py sending/editing/replying methods
Also, I would expand this snippet to be a very simple d.py or slash command so it's easier to immidiatly jump in and start playing with it

Reply:
i intentionally didnt mention if it was slash context or dpy context so the reader would realise that both could be used. But yeah ill make it explicit

as for expanding this into a command, im opposed to that. expanding it into a command would mean the user would just copy and paste the entire function. It would also mean the snippet is no longer just showing what the above text is trying to teach

Reply:
Added note about context, not resolving due to the unresolved discussion about expanding into a command